core/src/main/java/tc/oc/pgm/api/setting/SettingKey.java

```diff
@@ -57,6 +129,37 @@ public String getName() {
     return aliases.get(0);
   }
 
+  /**
+   * Get the display name of the {@link SettingKey}.
+   *
+   * @return The formatted display name.
+   */
+  public Component getDisplayName() {
+    return TranslatableComponent.of(displayNameKey, color, TextDecoration.BOLD);
+  }
+
+  /**
+   * Gets the lore of the setting.
+   *
+   * @param value the formatted value the setting is currently set to
+   * @return the complete lore text for the setting
+   */
+  public Component getLore(Component value) {
+    return TranslatableComponent.of(
+            SETTING_TRANSLATION_KEY + this.getName().toLowerCase() + ".description")
+        .args(value)
+        .color(TextColor.GRAY);
+  }
```

Comment:
Can we pre-compute the name and lore in the enum constructor? These won't change.